jupyter_scheduler/tests/test_execution_manager.py

```diff
@@ -1,50 +1,55 @@
-import os
-from contextlib import contextmanager
 from pathlib import Path
-from unittest.mock import PropertyMock, patch
 
 import pytest
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
+from conftest import DB_URL
 from jupyter_scheduler.executors import DefaultExecutionManager
-from jupyter_scheduler.orm import Base, Job
+from jupyter_scheduler.orm import Job
 
-NOTEBOOK_DIR = Path(__file__).resolve().parent / "test_staging_dir" / "job-3"
+JOB_ID = "69856f4e-ce94-45fd-8f60-3a587457fce7"
 NOTEBOOK_NAME = "side_effects.ipynb"
+SIDE_EFECT_FILE_NAME = "output_side_effect.txt"
+
+NOTEBOOK_DIR = Path(__file__).resolve().parent / "test_staging_dir" / "job-4"
 NOTEBOOK_PATH = NOTEBOOK_DIR / NOTEBOOK_NAME
-SIDE_EFFECT_FILE = NOTEBOOK_DIR / "output_side_effect.txt"
+SIDE_EFFECT_FILE = NOTEBOOK_DIR / SIDE_EFECT_FILE_NAME
 
 
-def test_execution_manager_with_side_effects():
-    db_url = "sqlite://"
-    engine = create_engine(db_url, echo=False)
-    Base.metadata.create_all(engine)
-    db_session = sessionmaker(bind=engine)
-    with db_session() as session:
+@pytest.fixture
+def load_job(jp_scheduler_db):
+    with jp_scheduler_db() as session:
         job = Job(
             runtime_environment_name="abc",
             input_filename=NOTEBOOK_NAME,
-            job_id="123",
+            job_id=JOB_ID,
```

Comment:
Instead of hard-coding the job ID, you should modify this fixture to return the ID of the job record it created. This is more similar to how jobs are created by the Scheduler backend.